CenterForOpenScience/osf.io
# [PLAT-973] Crossref is disabled with 'qatest' tag

## Purpose

Currently we are minting/syncing new DOI and DOI metadata for QA activities. This fix enables QA to disable syncing with the `qatest` tag.

## Changes

- Adds check for tag before instantiating the client
- Adds new test

## QA Notes

add the qatest tag to the node of your preprint, go to https://test.crossref.org/servlet/submissionAdmin?sf=search and see that it does not update itself to sync with the preprint. Remove the tag and see that updating resumes.

## Documentation

Not a user facing change.

## Side Effects

None that I know of.

## Ticket

https://openscience.atlassian.net/projects/PLAT/issues/PLAT-973
